src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/callback/AttachScannedDocumentsValidatorTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.callback;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.DuplicateDocsException;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Collections.singletonList;
+import static java.util.Collections.singletonMap;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+class AttachScannedDocumentsValidatorTest {
+
+    private static final AttachScannedDocumentsValidator VALIDATOR = new AttachScannedDocumentsValidator();
```

Comment:
After the class under test is a component setUp() method with Before annotation can be used

Reply:
I would use it if some mocked dependencies are needed which is not the case here so no need for extra lines of code just to initialise class for every test. It will not hold any state either